src/controller.coffee

```diff
@@ -827,8 +827,11 @@ Editor::redrawPalette = ->
 
     paletteBlockView.group.element.setAttribute 'data-id', entry.id
 
-    # enable tooltips
-    paletteBlockView.group.element.setAttribute('class', 'droplet-hover-div')
+    # enable tooltips, making sure not to overwrite any additional classes
+    # that may have been added (e.g. by tooltipster)
+    paletteBlockClass = paletteBlockView.group.element.getAttribute('class')
+    unless paletteBlockClass && paletteBlockClass.includes('droplet-hover-div')
+      paletteBlockView.group.element.setAttribute('class', 'droplet-hover-div')
```

Comment:
In our case, we only tooltipster this class by finding first via the `.droplet-hover-div` class, so there's no actual danger of clobbering it. If you think this reads poorly or sets us up badly for the future, I'm happy to make this change though.